test/unit/tracks/audio-track-list.test.js

```diff
@@ -48,14 +48,14 @@ QUnit.test('only one track is ever enabled', function(assert) {
   assert.equal(track2.enabled, false, 'track2 is disabled');
   assert.equal(track3.enabled, true, 'track3 is enabled');
 
-  track.enabled = true;
-  assert.equal(track.enabled, true, 'track is disabled');
-  assert.equal(track2.enabled, false, 'track2 is disabled');
+  track2.enabled = true;
+  assert.equal(track.enabled, false, 'track is disabled');
```

Comment:
We did not test to make sure that a track that is not enabled and already added, would be enabled correctly. Same with video tracks